frontend/ui/widget/inputtext.lua

```diff
@@ -664,6 +684,15 @@ function InputText:onKeyPress(key)
             handled = false
         end
     end
+    if not handled and Device:hasSymKey() then
+        local symkey = sym_key_map[key.key]
+        -- Do not match Shift + Sym + 'Alphabet keys'
+        if symkey and key.modifiers["Sym"] and not key.modifiers["Shift"] then
+            self:addChars(symkey)
+        else
+            handled = false
+        end
```

Comment:
You forgot the logical (as in the other branches) `handled = true` at the start of this one. Otherwise, this `handled=false` is a no-op.

Reply:
Am i not carrying it over from the one set before all the if statements?

Reply:
You're carrying it as `false` - so no need for the `else handled = false`, if you would set it to true in the `if simkey...`.
But the logic in all the other branches is to set it to `handled=true` early, and re-set it to false in the final else branch - so best to keep that same logic.